README.md

```diff
@@ -6,13 +6,252 @@ An Ansible role which installs and configures [Patroni](https://github.com/zalan
+
+### etcd
+
+Read more here: https://patroni.readthedocs.io/en/latest/SETTINGS.html#etcd
+
+- `patroni_etcd_host`: The host:port for the etcd endpoint. (**Default:** `""`)
+- `patroni_etcd_hosts`: List of etcd endpoint in format host1:port1,host2:port2,etc… Could be a comma separated string or an actual yaml list. (**Default:** `127.0.0.1:2379`)
+- `patroni_etcd_use_proxies`: If this parameter is set to true, Patroni will consider hosts as a list of proxies and will not perform a topology discovery of etcd cluster. (**Default:** `false`)
+- `patroni_etcd_url`: URL for the etcd. (**Default:** `""`)
+- `patroni_etcd_proxy`: Proxy url for the etcd. If you are connecting to the etcd using proxy, use this parameter instead of url. (**Default:** `""`)
+- `patroni_etcd_srv`: Domain to search the SRV record(s) for cluster autodiscovery. Patroni will try to query these SRV service names for specified domain (in that order until first success): _etcd-client-ssl, _etcd-client, _etcd-ssl, _etcd, _etcd-server-ssl, _etcd-server. (**Default:** `""`)
+- `patroni_etcd_protocol`:  (optional) http or https, if not specified http is used. If the `patroni_etcd_url` or `patroni_etcd_proxy` is specified - will take protocol from them. (**Default:** `http`)
+- `patroni_etcd_username`: (optional) username for etcd authentication. (**Default:** `""`)
+- `patroni_etcd_password`: (optional) password for etcd authentication. (**Default:** `""`)
+- `patroni_etcd_cacert`: (optional) The ca certificate. If present it will enable validation. (**Default:** `""`)
+- `patroni_etcd_cert`: (optional) file with the client certificate. (**Default:** `""`)
+- `patroni_etcd_key`: (optional) file with the client key. Can be empty if the key is part of cert. (**Default:** `""`)
+
+### etcdv3
+
+Read more here: https://patroni.readthedocs.io/en/latest/SETTINGS.html#etcdv3
+
+- `patroni_etcd_config_section`: If you want that Patroni works with Etcd cluster via protocol version 3, you need to use the `etcd3` section in the Patroni configuration file. All configuration parameters are the same as for `etcd`. (**Default:** `etcd`)
+
+### Consul
+
+Read more here: https://patroni.readthedocs.io/en/latest/ENVIRONMENT.html#consul
+
+- `patroni_consul_host`: The host:port for the Consul local agent. (**Default:** `"127.0.0.1:{{ patroni_consul_port | default(8500) }}"`)
+- `patroni_consul_port`: (optional) Consul port. (**Default:**  `8500`)
+- `patroni_consul_url`: URL for the Consul local agent, in format: http(s)://host:port. (**Default:** `""`)
+- `patroni_consul_scheme`: (optional) http or https, defaults to http. (**Default:** `http`)
+- `patroni_consul_token`: (optional) ACL token. (**Default:** `""`)
+- `patroni_consul_verify`: (optional) whether to verify the SSL certificate for HTTPS requests. (**Default:** `""`)
```

Comment:
It's a Boolean if I understand that right. So there has to be an implicit default.

Reply:
Which variable do you mean? Probably the `patroni_consul_verify`. Yes, that should be a bool. It's the way that the Jinja2 template is written why the variables are initialized but not set to anything. I will need to address it for all the other variables, too, to be consistent.